addons/web/static/src/core/debug/debug_menu_items.js

```diff
@@ -158,7 +114,6 @@ registry
     // Backend
     .add("runJSTestsItem", runJSTestsItem)
     .add("runJSTestsMobileItem", runJSTestsMobileItem)
```

Comment:
These are now available also in the frontend, so maybe update comment? Or do we *not* want them in the frontend?

Reply:
in master, they aren't in the frontend

Reply:
I checked with the website team, they do not want those items in the frontend so I have moved the definition of these items to the webclient.